src/data_table/local_data_table.js

```diff
@@ -480,6 +526,8 @@ class LocalDataTable extends View {
   }
 
   _dataTableConfig() {
+    let displayStart = this._parsePageNumberFromQueryString() * this.paginateLength;
+    let recordTotal = displayStart + 10;
```

Comment:
this 10 needs to be `this.paginateLength` instead

Reply:
✅